### PR TITLE
Resolve conflicts when dropping T prefix in /lib/wrappers/libuv.nim

### PR DESCRIPTION
**Pull request by Perelandric**
Additional fix for https://github.com/nim-lang/Nim/issues/2773.

`HandleType` enum variants now have `ht` prefix and `T` is dropped from types that previously caused conflict.

Old `T` types are deprecated but can't deprecate old enum variants, so this is a breaking change and it's noted in `web/news.txt`.

Tests pass on Linux.

### CODE REVIEW

File: lib/wrappers/libuv.nim [hunk 1]
```diff
@@ -60,8 +60,9 @@ type
     EAINONAME, EAISERVICE, EAISOCKTYPE, ESHUTDOWN, EEXIST
 
   HandleType* {.size: sizeof(cint).} = enum
-    UNKNOWN_HANDLE = 0, TCP, UDP, NAMED_PIPE, TTY, FILE, TIMER, PREPARE, CHECK,
-    IDLE, ASYNC, ARES_TASK, ARES_EVENT, PROCESS, FS_EVENT
+    htUnknownHandle = 0, htTcp, htUdp, htNamedPipe, htTty, htFile, htTimer,
```
Comment (dom96) [inline lib/wrappers/libuv.nim]: Maybe it would be a better idea to just make this enum ``{.pure.}``?

File: lib/wrappers/libuv.nim [hunk 2]
```diff
@@ -85,14 +86,14 @@ type
...
-  TFsEvent* {.pure, final, importc: "uv_fs_event_t", header: "uv.h".} = object
+  FsEvent* {.pure, final, importc: "uv_fs_event_t", header: "uv.h".} = object
     loop* {.importc: "loop".}: PLoop
     typ* {.importc: "type".}: HandleType
     close_cb* {.importc: "close_cb".}: CloseProc
     data* {.importc: "data".}: pointer
     filename {.importc: "filename".}: cstring
 
-  PFsEvent* = ptr TFsEvent
+  PFsEvent* = ptr FsEvent
```
Comment (dom96) [inline lib/wrappers/libuv.nim]: What about these ``P`` prefixes? Those should be turned into a ``Ptr`` suffix.

**Comment by dom96** [comment]
@Perelandric Truly appreciate all this work you're doing. Surprised you are not bored of it yet!

File: lib/wrappers/libuv.nim (hunk 1 above)
Comment (Perelandric) [inline lib/wrappers/libuv.nim]: Could do that. I'd like it if most enums were `{.pure.}`, but that's just me.

**Comment by Perelandric** [comment]
@dom96 Thanks, until I dig into the guts of Nim a bit more, this is the easiest way for me to contribute. :-)

File: lib/wrappers/libuv.nim (hunk 2 above)
Comment (Perelandric) [inline lib/wrappers/libuv.nim]: Wondered about those. The original issue was just for the `T` prefixes so I thought maybe there'd be a separate issue on its way. Either way, I can work on those as well this week.

**Pull request closed without merging**